ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/DivulgenceIT.scala

```diff
@@ -327,10 +324,10 @@ class DivulgenceIT
     }
   }
 
-  "should not expose divulged contracts in ACS" in allFixtures { ctx =>
+  it should "not expose divulged contracts in ACS" in allFixtures { ctx =>
     val wfid = testIdsGenerator.testWorkflowId("divulgence-test-acs-workflow-id")
     for {
-      beforeTest <- transactionClient(ctx).getLedgerEnd.map(_.getOffset)
+      _ <- transactionClient(ctx).getLedgerEnd.map(_.getOffset)
```

Comment:
If this is not being used, we don't need to look it up in the first place.

Reply:
Makes sense. I'll clean this up.

Reply:
Addressed by 8a91173bd